community/kernel/src/main/java/org/neo4j/unsafe/batchinsert/BatchInserterImpl.java

```diff
@@ -285,7 +284,8 @@ public Label apply( long from )
         Token[] types = getRelationshipTypeStore().getTokens( Integer.MAX_VALUE );
         relationshipTypeTokens = new BatchTokenHolder( types );
         indexStore = life.add( new IndexConfigStore( this.storeDir, fileSystem ) );
-        schemaCache = new SchemaCache( neoStore.getSchemaStore() );
+        // TODO: perhaps we should differentiate for BatchInserter running in enterprise here...
+        schemaCache = new SchemaCache( new StandardConstraintSemantics(), neoStore.getSchemaStore() );
```

Comment:
Let's defer creating PEC in BatchInserter, which is intended for high-speed creation of known-good data.